andela/the_spinners-backend
# #169817564 Users receive comments notifications

#### What does this PR do?
- Has receive comments notifications endpoint functionality
- Has receive comments notifications test
#### Description of Task to be completed?
- When a requester comment on a trip request a manager should get notification
- When a manager comment on a trip request a requester should get notification
#### How should this be manually tested?
- Clone the repos
- Using a terminal change directory to the cloned repos
- Run `npm install`
- Run `npm run dev`
- Create a user and assign him to a manger
- Login the user and create a trip with this route `http://localhost:3000/api/trips/one-way`
- Create a comment on the created trip using this route `http://localhost:3000/api/trips/requests/:tripId/comments`
- When a comment was created a user should receive a notification in-app or in email
#### What are the relevant pivotal tracker stories?
[#169817564](https://www.pivotaltracker.com/story/show/169817564)
#### Screenshots
<img width="1044" alt="Screen Shot 2020-01-22 at 16 48 01" src="https://user-images.githubusercontent.com/38500196/72905594-72ff8500-3d39-11ea-9628-f5c08e4e64f1.png">

<img width="1168" alt="Screen Shot 2020-01-22 at 16 48 30" src="https://user-images.githubusercontent.com/38500196/72905616-7f83dd80-3d39-11ea-8dfe-1d5cbb595c4a.png">
